mw-webapp/src/pages/reportsTable/ReportsTable.tsx

```diff
@@ -0,0 +1,45 @@
+import {useMemo} from "react";
+import {getCoreRowModel, HeaderGroup, RowModel, useReactTable} from "@tanstack/react-table";
+import {columns} from "src/component/table/columns";
+import {Table} from "src/component/table/Table";
+import {DayReport} from "src/model/businessModel/DayReport";
+import {useGetDataTable} from "src/service/TableService";
+
+/**
+ * Generated table data of {@link DayReport[]}
+ */
+export interface TableReports {
+  /**
+ * Group headers table
+ */
+  headerGroup: HeaderGroup<DayReport>[];
+  /**
+ * Object rows table
+ */
+  rowModel: RowModel<DayReport>;
+}
+
+/**
+ * Wrapper for receiving {@link DayReport[]} data and transferring it to table
```

Comment:
It's not a wrapper by perspective of this comment. Lets talk about it on next meeting, ask me to tell about external and internal things

Reply:
Done